### DISCUSSION TITLE
Respawn Instance

### DISCUSSION THREAD

**Discussion by thesamgordon** [Minestom/Minestom#1424] [Help]
Hi,

Every time I hit the respawn button from an instance with a flat world, I get sent to a void world. The instance they respawn in is the exact same. Why is it void?

`
        globalEventHandler.addListener(PlayerLoginEvent.class, event -> {
            final Player player = event.getPlayer();
            event.setSpawningInstance(instanceContainer);
            player.setRespawnPoint(new Pos(0, 42, 0));
        });
`

https://pneumonoultramicroscopicsilicovolcanoconiosis.club/jUlqUqxq

**Comment by thesamgordon** [comment]
After further investigation, it just looks like the blocks don't exist on that client. Other players would see them falling through the floor.

**Reply by Timofffee** [reply]
And the solution was found?

**Reply by thesamgordon** [reply]
> And the solution was found?

Nope.
